packages/date-picker/test/form-input.test.js

```diff
@@ -220,13 +218,25 @@ describe('form input', () => {
 
     it('should fail validation without value', () => {
       expect(datePicker.checkValidity()).to.be.false;
-      expect(datePicker.validate()).to.be.false;
```

Comment:
I think the `validate()` assertions don't make sense since the component doesn't have custom implementation of `validate()`.